package.json

```diff
@@ -896,6 +927,14 @@
           "description": "Status bar color when in EasyMotionInput mode.",
           "default": "#007acc"
         },
+        "vim.statusBarColors.sneaklabelinputmode": {
```

Comment:
`sneaklabelinputmode` should be camel case

Reply:
I am looking at the other settings for the status bar colors and they all follow the lowercase mode. Should I still change it to camelCase?